crow/reference.md

```diff
@@ -70,6 +70,28 @@ output[n].volts = 2.0 -- tell output n to move toward 2.0 volts, over the slew t
 v = output[n].volts   -- set v to the instantaneous voltage of output n
 ```
 
+### shaping cv
+
+*since v1.0.3*
+
+```lua
+output[n].shape = 'expo' -- travel to the .volts destination with a non-linear path
```

Comment:
literally `e` will work because because of how i built the C parser (optimized for SPEED), so whichever feels more natural (definitely state 'expo' at the minimum)